checker/jdk/index/src/java/lang/String.java

```diff
@@ -635,6 +635,7 @@ public String(StringBuilder builder) {
      *
      * @since 1.6
      */
+    @SuppressWarnings("contracts.conditional.postcondition.not.satisfied") // values.length != 0 => this is @MinLen(1 as values.length is @LengthOf("this")
```

Comment:
Please add a comment explaining what conditional postcondition is not satisfied.  I don't see one here and the existing comment is not sufficient for me.